src/OrbitGl/TrackManager.cpp

```diff
@@ -375,6 +375,32 @@ bool TrackManager::GetTrackTypeVisibility(Track::Type type) const {
   return track_type_visibility_.at(type);
 }
 
+Track* TrackManager::GetTrackFromTimerInfo(const TimerInfo& from) {
+  switch (from.type()) {
+    case TimerInfo::kNone:
+    case TimerInfo::kApiScope:
+      return GetOrCreateThreadTrack(from.thread_id());
+    case TimerInfo::kCoreActivity:
+      return GetOrCreateSchedulerTrack();
+    case TimerInfo::kFrame:
+      return GetOrCreateFrameTrack(capture_data_->instrumented_functions().at(from.function_id()));
+    case TimerInfo::kGpuActivity:
+    case TimerInfo::kGpuCommandBuffer:
+    case TimerInfo::kGpuDebugMarker:
+      return GetOrCreateGpuTrack(from.timeline_hash());
+    case TimerInfo::kApiScopeAsync:
+      return GetOrCreateAsyncTrack(from.api_scope_name());
+    case TimerInfo::kSystemMemoryUsage:
+      return GetSystemMemoryTrack();
+    case TimerInfo::kCGroupAndProcessMemoryUsage:
+      return GetCGroupAndProcessMemoryTrack();
+    case TimerInfo::kPageFaults:
+      return GetPageFaultsTrack();
+    default:
+      return nullptr;
```

Comment:
Instead of using `default:`, I'd suggest using:

```
    case orbit_client_protos::TimerInfo_Type_TimerInfo_Type_INT_MIN_SENTINEL_DO_NOT_USE_:
    case orbit_client_protos::TimerInfo_Type_TimerInfo_Type_INT_MAX_SENTINEL_DO_NOT_USE_:
      UNREACHABLE();
```

and explicitly adding a case for `kApiEvent`. This way, if we add a new type, the compiler will force us to add an entry here.

Reply:
done.